go.mod

```
@@ -181,19 +181,18 @@ require (
 	github.com/multiformats/go-multistream v0.4.1 // indirect
 	github.com/multiformats/go-varint v0.0.7 // indirect
 	github.com/onsi/ginkgo/v2 v2.11.0 // indirect
-	github.com/opencontainers/runtime-spec v1.0.2 // indirect
+	github.com/opencontainers/runtime-spec v1.1.0 // indirect
 	github.com/openzipkin/zipkin-go v0.4.1 // indirect
 	github.com/petar/GoLLRB v0.0.0-20210522233825-ae3b015fd3e9 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/polydawn/refmt v0.89.0 // indirect
 	github.com/prometheus/client_model v0.4.0 // indirect
 	github.com/prometheus/common v0.44.0 // indirect
-	github.com/prometheus/procfs v0.11.0 // indirect
+	github.com/prometheus/procfs v0.11.1 // indirect
 	github.com/prometheus/statsd_exporter v0.22.7 // indirect
 	github.com/quic-go/qpack v0.4.0 // indirect
-	github.com/quic-go/qtls-go1-19 v0.3.3 // indirect
-	github.com/quic-go/qtls-go1-20 v0.2.3 // indirect
-	github.com/quic-go/quic-go v0.36.4 // indirect
+	github.com/quic-go/qtls-go1-20 v0.3.2 // indirect
```

Not required, but nice to have, see https://github.com/quic-go/qtls-go1-20/releases/tag/v0.3.3.

```suggestion
	github.com/quic-go/qtls-go1-20 v0.3.3 // indirect
```

---

I think the go toolchain would remove it from `go mod vendor` anyway, but hey why not ?

---

Why would it? This is an important input to minimum version selection.

---

```
ls vendor/github.com/quic-go/qtls-go1-20/
LICENSE    cipher_suites.go           handshake_messages.go      notboring.go  tls.test
README.md  common.go                  handshake_server.go        prf.go        unsafe.go
alert.go   conn.go                    handshake_server_tls13.go  quic.go
auth.go    handshake_client.go        key_agreement.go           ticket.go
cache.go   handshake_client_tls13.go  key_schedule.go            tls.go
```
ok nvm I'm wrong.

---

Bumped it